### PR TITLE
Make the consumption for charges the delete button

### PR DESCRIPTION
**Pull request by LinconOssa**
The functionality of eliminating charges is made. This functionality only works with the following:
image

Since the other fields in the postman don't let me either, it appears like this:
image
image

### CODE REVIEW

File: src/services/positions/deletePositions/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    let data;
+    try {
+      data = await res.json();
+    } catch (error) {
+      throw new Error("Failed to parse response JSON");
+    }
+
+    if (!res.ok) {
+      const errorMessage = `Error al eliminar el Caso de Uso. Status: ${
```
Comment (SelsaErojas) [inline src/services/positions/deletePositions/index.tsx:36]: you should change the error message according to the funcionality

File: src/services/positions/deletePositions/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        res.status
+      }, Data: ${JSON.stringify(data)}`;
+      throw new Error(errorMessage);
+    }
+    return data;
+  } catch (error) {
+    console.error("Failed to delete Caso de Uso:", error);
```
Comment (SelsaErojas) [inline src/services/positions/deletePositions/index.tsx:43]: you should change the error message according to the funcionality

**Review by SelsaErojas — CHANGES_REQUESTED** [review]

File: src/services/positions/deletePositions/index.tsx (hunk 1 above)
Comment (LinconOssa) [reply src/services/positions/deletePositions/index.tsx]: Resolved

File: src/services/positions/deletePositions/index.tsx (hunk 2 above)
Comment (LinconOssa) [reply src/services/positions/deletePositions/index.tsx]: Resolved

**Review by SelsaErojas — APPROVED** [review]
Functional

**Review by Dlfp — APPROVED** [review]

**Pull request merged**